Tests/Generator/BundleGeneratorTest.php

```diff
@@ -17,7 +17,7 @@ class BundleGeneratorTest extends GeneratorTest
 {
     public function testGenerateYaml()
     {
-        $this->getGenerator()->generate('Foo\BarBundle', 'FooBarBundle', $this->tmpDir, 'yml', false);
+        $this->getGenerator()->generate('Foo\BarBundle', 'FooBarBundle', $this->tmpDir, 'yml', false, true);
```

Comment:
you can keep the previous version if you define it default to ```true```, otherwise its a breaking change for users who implement the generate()-method somewhere else